### PR TITLE
oxycards: init at unstable-2023-03-11

### PR DESCRIPTION
**Pull request by ghost**
## Description of changes

Oxycards is a quiz card application built within the terminal.

> https://github.com/BrookJeynes/oxycards

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by UnidealisticRaccoon** [comment]
Result of `nixpkgs-review pr 279663` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>oxycards</li>
  </ul>
</details>

**Review by GetPsyched — APPROVED** [review]
LGTM

File: pkgs/by-name/ox/oxycards/package.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  };
+
+  cargoHash = "sha256-0mA5OAWL8pNP3riu2lBmmvMtYIBJH6eNX3Ibp5ikNj4=";
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  meta = with lib; {
+    description = "Oxycards is a quiz card application built within the terminal";
+    homepage = "https://github.com/BrookJeynes/oxycards";
+    license = licenses.mit;
```
Comment (GetPsyched) [inline pkgs/by-name/ox/oxycards/package.nix:25]: Uh, there is no license upstream? This should be set to `licenses.unfree` in that case.

**Review by GetPsyched — CHANGES_REQUESTED** [review]
Oops, missed this before.

File: pkgs/by-name/ox/oxycards/package.nix (hunk 1 above)
Comment (UnidealisticRaccoon) [reply pkgs/by-name/ox/oxycards/package.nix:25]: **Thanks for the review!**

As for the License, I got the `MIT` license from the [Cargo.toml](https://github.com/BrookJeynes/oxycards/blob/main/Cargo.toml#L6).

Comment (GetPsyched) [reply pkgs/by-name/ox/oxycards/package.nix:25]: That is so weird lol. I guess this is fine then.

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -19199,6 +19199,16 @@
     githubId = 17836748;
     name = "Mason Mackaman";
   };
+  ursuslotor = {
```
Comment (GetPsyched) [inline maintainers/maintainer-list.nix:19202]: It's recommended that this matches your GitHub username, BTW.
https://github.com/NixOS/nixpkgs/blob/cf53751a16df6ae52eb3be7019aa9c34017e490b/maintainers/maintainer-list.nix#L34

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (UnidealisticRaccoon) [reply maintainers/maintainer-list.nix:19202]: If it's a must, I could change it, but it it's alright, I would like to keep it shorter using `ursuslotor`. This handle is also used in my [email](mailto:me@ursuslotor.xyz), domain, and [other sites](https://gravatar.com/ursuslotor), like [keybase](https://keybase.io/ursuslotor).

P.S. I could put an `aka` in my Github bio if that would suffice.

File: pkgs/by-name/ox/oxycards/package.nix (hunk 1 above)
Comment (UnidealisticRaccoon) [reply pkgs/by-name/ox/oxycards/package.nix:25]: I might open a PR upstream suggesting the addition of a `LICENSE` file.

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (UnidealisticRaccoon) [reply maintainers/maintainer-list.nix:19202]: Oh, just saw it. There is already an `@ursuslotor` on [Github](https://github.com/ursuslotor), so indeed, it's better if I change the handle.

**Review by GetPsyched — APPROVED** [review]

**Pull request closed without merging**